### PR TITLE
test($sanitize): skip clobber test on Edge

### PR DESCRIPTION
**Pull request by Narretz**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?**

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our [guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [ ] Fix/Feature: [Docs](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#documentation) have been added/updated
- [ ] Fix/Feature: Tests have been added; existing tests pass

**Other information**:

### CODE REVIEW

File: test/ngSanitize/sanitizeSpec.js [hunk 1]
```diff
@@ -246,25 +246,28 @@ describe('HTML', function() {
       .toEqual('<p>text1text2</p>');
   });
 
-  it('should throw on clobbered elements', function() {
-    inject(function($sanitize) {
-      expect(function() {
-        $sanitize('<form><input name="parentNode" /></form>');
-      }).toThrowMinErr('$sanitize', 'elclob');
+  if (!/Edge\/16/.test(window.navigator.userAgent)) {
+    // Skip test on Edge 16 due to browser bug.
```
Comment (petebacondarwin) [inline test/ngSanitize/sanitizeSpec.js]: Perhaps include this link here: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/15263856/#pagenotfound

Comment (Narretz) [reply test/ngSanitize/sanitizeSpec.js]: Problem is that the issue is private, so it goes to 404. Maybe we should create an issue in our repo with a short description?

**Review by petebacondarwin — COMMENTED** [review]
It just occurred to me that on Edge, when this fails to fail, the return value from `$sanitize` appears to be something like the console object.
Am I right?
How about we change the code so that if the result that will be returned from `$sanitize` is not a string then we throw the `elclob` error?
That way we don’t need to change our tests and we are covered from returning something dodgy in Edge.

**Review by mgol — APPROVED** [review]
`parentNode` is buggy in Edge as well but if our tests pass then I guess we may leave it as it is. LGTM.

**Review by mgol — APPROVED** [review]
LGTM but the `config.browserNoActivityTimeout` change should perhaps land in a separate commit.
